R/oc_process.R

```diff
@@ -246,18 +245,38 @@ oc_build_url <- function(query_par, endpoint) {
 
 #' GET request from OpenCage
 #'
-#' @param oc_url character string URL with query parameters, built with
+#' @param oc_url_parts list with path and query, built with
 #'   `oc_build_url()`
 #'
-#' @return crul::HttpResponse object
+#' @return httr2 response
 #' @noRd
 
-oc_get <- function(oc_url) {
-  client <- crul::HttpClient$new(
-    url = oc_url,
-    headers = list(`User-Agent` = oc_ua_string)
-  )
-  client$get()
+oc_get <- function(oc_url_parts = NULL) {
+
+  query_req <- build_query_with_req(oc_url_parts)
+
```

Comment:
```suggestion
oc_get <- function(query_req = NULL) {
```

The `build_query_with_req()` in `.oc_process()` ought to suffice?